.bazelrc

```diff
@@ -136,9 +135,6 @@ build:remote --extra_execution_platforms=@npm//@angular/build-tooling/bazel/remo
 build:remote --host_platform=@npm//@angular/build-tooling/bazel/remote-execution:platform_with_network
 build:remote --platforms=@npm//@angular/build-tooling/bazel/remote-execution:platform_with_network
 
-# Set remote caching settings
-build:remote --remote_accept_cached=true
```

Comment:
This is the default. IMO only setting it (to the default) on `--config=remote` is confusing...